src/collection-browser.ts

```diff
@@ -618,6 +618,9 @@ export class CollectionBrowser
         @titleLetterChanged=${this.titleLetterSelected}
         @creatorLetterChanged=${this.creatorLetterSelected}
       >
+        <div slot="sortbar-left-slot">
+          <slot name="sortbar-left-slot"></slot>
+        </div>
```

Comment:
Q: Is the wrapper div needed here? Could we just do `<slot name="sortbar-left-slot" slot="sortbar-left-slot">` directly?